pkg/coscheduling/coscheduling.go

```diff
@@ -256,7 +273,7 @@ func (cs *Coscheduling) Permit(ctx context.Context, state *framework.CycleState,
 		klog.V(3).Infof("The count of podGroup %v/%v/%v is not up to minAvailable(%d) in Permit: current(%d)",
 			pod.Namespace, podGroupName, pod.Name, minAvailable, current)
 		// TODO Change the timeout to a dynamic value depending on the size of the `PodGroup`
-		return framework.NewStatus(framework.Wait, ""), 10 * PermitWaitingTime
+		return framework.NewStatus(framework.Wait, ""), 10 * cs.args.PermitWaitingTime
```

Comment:
I suggest deleting `10 * ` here.

Reply:
+1. 
And changing the default value into 10s on line 405. 
params.PermitWaitingTime = "10s"

Reply:
+1

Reply:
it's a mistake, corrected it.

Reply:
> +1.
> And changing the default value into 10s on line 405.
> params.PermitWaitingTime = "10s"

The current default value is 30s.

Reply:
The current PermitWaitingTime is 1s. It's better to change it to 10s.

Reply:
changed the default to 10s.